pennylane_aqt/ops.py

```diff
@@ -68,5 +67,4 @@ class MS(Operation):
     num_params = 1
     num_wires = 2
     par_domain = "R"
-    grad_method = None
-    grad_recipe = None
+    grad_method = "A"
```

Comment:
Ah, so the MS gate supports the default parameter-shift in the end?

Reply:
Looks like it, yes